app/Http/Controllers/Api/TracksController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Album;
+use App\Track;
+use App\Reciter;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+
+class TracksController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Reciter $reciter, Album $album)
+    {
+        $tracks = Track::where('album_id', $album->id)->get();
+
+        return $tracks;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request, Reciter $reciter, Album $album)
+    {
+        $track = new Track;
+        $track->name = $request->get('name');
+        $track->slug = str_slug($request->get('name'));
+        $track->album_id = $album->id;
+        $track->mp3_link = $request->get('mp3_link');
+        $track->file_path = $request->get('file_path');
+        $track->hits = 1;
+        $track->track_number = $request->get('track_number');
+        $track->language = 'English';
```

Comment:
We should probably use standard language codes for this. https://en.wikipedia.org/wiki/ISO_639-1